### PR TITLE
Add missing, and update existing, documentation on the RSA classes

### PR DESCRIPTION
**Pull request by bartonjs**
cc: @carlossanlop

### CODE REVIEW

File: xml/System.Security.Cryptography/RSACng.xml [hunk 1]
```diff
@@ -861,14 +922,36 @@
...
+        <param name="hash">The hash value of the data to be signed.</param>
         <param name="destination">To be added.</param>
-        <param name="hashAlgorithm">To be added.</param>
-        <param name="padding">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="hashAlgorithm">The hash algorithm used to create the hash value of the data.</param>
+        <param name="padding">The padding.</param>
```
Comment (carlossanlop) [inline xml/System.Security.Cryptography/RSACng.xml]: This parameter could use a better description.

File: xml/System.Security.Cryptography/RSAOpenSsl.xml [hunk 2]
```diff
@@ -448,12 +556,26 @@
...
       </Parameters>
       <Docs>
-        <param name="hash">To be added.</param>
-        <param name="hashAlgorithm">To be added.</param>
-        <param name="padding">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="hash">The hash value of the data to be signed.</param>
+        <param name="hashAlgorithm">The hash algorithm used to create the hash value of the data.</param>
+        <param name="padding">The padding.</param>
```
Comment (carlossanlop) [inline xml/System.Security.Cryptography/RSAOpenSsl.xml]: This parameter could use a better description.

File: xml/System.Security.Cryptography/RSAOpenSsl.xml [hunk 3]
```diff
@@ -481,13 +603,18 @@
...
-        <param name="hash">To be added.</param>
-        <param name="signature">To be added.</param>
-        <param name="hashAlgorithm">To be added.</param>
-        <param name="padding">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="hash">The hash value of the signed data.</param>
+        <param name="signature">The signature data to be verified.</param>
+        <param name="hashAlgorithm">The hash algorithm used to create the hash value.</param>
+        <param name="padding">The padding mode.</param>
```
Comment (carlossanlop) [inline xml/System.Security.Cryptography/RSAOpenSsl.xml]: This parameter has one extra word, but I think it could also be described better.

Comment (carlossanlop) [reply xml/System.Security.Cryptography/RSAOpenSsl.xml]: Same suggestion applies to the rest of the padding params.

Comment (bartonjs) [reply xml/System.Security.Cryptography/RSAOpenSsl.xml]: The "padding" parameter descriptions are the ones that are already present on the documented overloads and variants (e.g. https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.rsa.signdata?view=netframework-4.8).

They can be long winded, such as "The padding mode to use for the {encryption|decryption|signing|verification} process.", but there's really nothing else that I can think of to say about them.  I'm open to suggestions, but absent a specific suggestion I'm going to leave them as-is.

**Comment by bartonjs** [comment]
Looks like there were some bad cref/xrefs, I'll get them after lunch (assuming no one does it before then).

**Comment by BillWagner** [comment]
@bartonjs This now has merge conflicts. Can you resolve those?

**Comment by bartonjs** [comment]
@BillWagner Conflicts resolved, bad crefs/xrefs resolved.

**Pull request merged**